python/src/trezorlib/cli/benfen.py

```
@@ -0,0 +1,90 @@
+    address: str,
+    message: str,
+) -> dict:
+    """Sign message with Benfen address."""
+    address_n = tools.parse_path(address)
+    ret = benfen.sign_message(client, address_n, message)
+    output = {
+        "message": message,
+        "address": ret.address,
+        "signature": ret.signature.hex(),
+    }
+    return output
```

Ruff (0.8.2)

Unnecessary assignment to `output` before `return` statement

Remove unnecessary assignment

(RET504)